recipe/run_test.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 
-set -e
+set -ex
 
-${PYTHON} -c "import dpctl"
-python -m pytest -q -ra --disable-warnings --cov dpctl --cov-report term-missing --pyargs dpctl -vv
```

Comment:
why using here system's python?  we should use here ${PYTHON} too, shouldn't we?